docs/rpc/components/schemas/get-transaction.schema.yaml

```
@@ -14,3 +14,9 @@ properties:
   result:
     type: string
     description: Transaction execution result (Clarity value)
+  block_height:
+    type: [number, "null"]
```

nit: shouldn't this be `integer` and not `number`?

```suggestion
    type: [integer, "null"]
```